starters/serverless-framework-sqs-dynamodb/src/models/technology/destroy.ts

```diff
@@ -0,0 +1,9 @@
+import { removeFromCache } from '@/utils/cache/removeFromCache';
+import { deleteItem } from '@/utils/dynamodb/deleteItem';
+import { getCacheKey } from './getCacheKey';
+
+export const destroy = async (key: string) => {
+	const item = await deleteItem(process.env.TECHNOLOGIES_TABLE, { id: key });
```

Comment:
does the `deleteItem` method return the deleted entry? 🤔 

hat's what the unit test validates, but it feels strange for me after postgresql, where delete did not return anything, just information about how many rows were affected and the rest endpoint returns with OK.

Reply:
My implementation has chosen to return the record. It doesn't necessarily have to do this, but DynamoDB does return it, therefore, I do.

Reply:
Understood, I just wanted confirmation on the dynamodb implementation.